paperqa/clients/crossref.py

```diff
@@ -71,35 +72,6 @@
     "source_quality": {"container-title"},
     "doc_id": {"DOI"},
 }
-CROSSREF_CONTENT_TYPE_TO_BIBTEX_MAPPING: dict[str, str] = {
```

Comment:
Jw, why remove this?

Reply:
This dictionary is also needed by openalex, so I just moved it to more generic file